SPECS/grep/grep.spec

```diff
@@ -1,31 +1,44 @@
 Summary:        Programs for searching through files
 Name:           grep
 Version:        3.1
-Release:        4%{?dist}
+Release:        5%{?dist}
 License:        GPLv3+
-URL:            http://www.gnu.org/software/grep
-Group:          Applications/File
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+Group:          Applications/File
+URL:            http://www.gnu.org/software/grep
 Source0:        http://ftp.gnu.org/gnu/grep/%{name}-%{version}.tar.xz
+Patch0:         grep-3.31-man-fix-gs.patch
+Patch1:         grep-3.31-help-align.patch
+Patch2:         grep-3.1-glibc-2.28-fix.patch
+BuildRequires:  pcre-devel
+Requires:       pcre
```

Comment:
It looks like `grep` is part of chroot manifest, so we should now also add `pcre-8.44-1.cm1.x86_64.rpm` into it. We currently only have `pcre-libs` there.

Reply:
good point. pcre is only required for regular perl expression support, but this should be added. will update